syntax = "proto3";

option csharp_namespace = "PLANetary.Communication.Protobuf";

import "identifier.proto";

message QueryParentSel {
   QueryId queryId = 1;
   NodeId childId = 2;
}